Dockerfile

```diff
@@ -1,19 +1,29 @@
 FROM quay.io/ukhomeofficedigital/nodejs-base:v4.4.2
 
-COPY . /app
-
-RUN npm install -g npm@3
+RUN mkdir /public
 
 RUN yum clean all && \
   yum update -y && \
   yum install -y git && \
-  yum clean all && \
-  rpm --rebuilddb && \
-  rm -rf node_modules && \
-  npm --production=false install --unsafe-perm --no-optional && \
-  npm test && \
-  npm prune --production && \
-  chown -R nodejs:nodejs .
+  yum clean all
+
+RUN rpm --rebuilddb
+
+RUN npm install -g npm@3
+
+COPY package.json /tmp/package.json
+
+RUN cd /tmp && npm install --production --unsafe-perm --no-optional
+
+RUN mkdir -p /app && cp -a /tmp/node_modules /app/
```

Comment:
is this final, i thought you were going to `mkdir /app && cd /app && npm i` before copying everything else into app?